Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/dungeon_items_weapons.yml

```diff
@@ -272,6 +272,7 @@
     - WeaponCaseLongHristovExpedition
     - WeaponCaseShortSvalinnExpedition
     - WeaponCaseShortAdvancedLaserExpedition
+    - WeaponCaseShortTheArbitrator
```

Comment:
With weapon stats as they are right now, this weapon belongs to T4 minimum, maybe even to T5.

Reply:
The reason why the Stats were chosen was because it was post to be a high obtainable Civilian weapon you get in expeditions.  When speaking with checkraze in the discord he brought up the stats of the Krammar pretty much said be smiliar/close to that.   So I made it a 4 round, 2 firing rate revolver

Reply:
> a high obtainable
Not sure if it should be easily available. Wide availability of sawn-off db shotgun is problematic (the weapon is too good and too easy to obtain, this tbh has to be addressed sometime), Arbitrator is a sawn-off db shotgun but better: holds twice as much shots.

Reply:
that was a miss type. it was posted to be a "high-tier obtainable civilian weapon" it is post to be an upgrade to the krammar and to the sawnoff

Reply:
Right, so looks like Kammerer is in T2, so I guess having this gun in T3 is fine. Can always shuffle it different category afterwards.